### DISCUSSION TITLE
Wake on LAN doesn't work on Ubuntu MATE 16.04

### DISCUSSION THREAD

**Discussion by fottsia** [epoptes/epoptes#27] [Q&A]
Hi, I tried the WOL function of Epoptes in Ubuntu MATE 16.04 and it's not working.
It was working with the same client in Ubuntu 12.04 with Gnome Flashback.

**Comment by alkisg** [comment]
This is a problem with the tlp package, not with epoptes.
The ubuntu-mate-desktop metapackage depends on tlp. The tlp package disables WOL in order to lower power consumption.
Since tlp can't be uninstalled without also removing ubuntu-mate-desktop, you can disable it by editing _one_ of the following two lines in /etc/default/tlp:

> \# Set to 0 to disable, 1 to enable TLP.
> TLP_ENABLE=0
> \# Disable wake on LAN: Y/N
> WOL_DISABLE=N

Fortunately, ubuntu-mate-desktop no longer depends on tlp on newer versions, so this only affects Ubuntu MATE 16.04.

**Comment by 13guter31** [comment]
Hello. To wake up (WOL), in what mode should I configure the network card?

**Comment by alkisg** [comment]
It's automatically set to "g" by epoptes-client, you don't need to do it manually.

**Comment by 13guter31** [comment]
It's clear. Thank you.
